tests/spec/w3c/defaults-spec.js

```diff
@@ -51,6 +52,7 @@ describe("W3C — Defaults", () => {
       "check-punctuation": false,
       "fake-linter-rule": "foo",
       "check-internal-slots": true,
+      "check-charset": true,
```

Comment:
@marcoscaceres should this be set to `true` ? the karma test failed for this case ...

Reply:
This one should be false, to match core/defaults.

Reply:
@marcoscaceres done, sent a PR!